include/openssl/ssl.h

```diff
@@ -5115,22 +5115,34 @@ OPENSSL_EXPORT int SSL_CTX_set1_sigalgs_list(SSL_CTX *ctx, const char *str);
 // more convenient to codesearch for specific algorithm values.
 OPENSSL_EXPORT int SSL_set1_sigalgs_list(SSL *ssl, const char *str);
 
-// SSL_CTX_get_security_level returns 3. This is only to maintain compatibility
+// SSL_CTX_get_security_level returns 0. This is only to maintain compatibility
 // with OpenSSL and no security assumptions should be based on the number this
 // function returns.
 //
-// Per OpenSSL's definition of Level 3:
-// Level 3:
-// Security level set to 128 bits of security. As a result RSA, DSA and DH keys
-// shorter than 3072 bits and ECC keys shorter than 256 bits are prohibited. In
-// addition to the level 2 exclusions cipher suites not offering forward secrecy
-// are prohibited. TLS versions below 1.1 are not permitted. Session tickets are
-// disabled.
-//
-// AWS-LC only supports atl least 128 bits of security, but we don't directly
-// prohibit session tickets and TLS 1.0 like Level 3 in OpenSSL states. This
-// behavior may change if we're asked to support actual Security level setting
-// in AWS-LC.
+// Per OpenSSL's definition of Level 0, 1, and 2:
+// Level 0:
+// Everything is permitted. This retains compatibility with previous versions of
+// OpenSSL.
+//
+// Level 1
+// The security level corresponds to a minimum of 80 bits of security. Any
+// parameters offering below 80 bits of security are excluded. As a result RSA,
+// DSA and DH keys shorter than 1024 bits and ECC keys shorter than 160 bits are
+// prohibited. All export cipher suites are prohibited since they all offer less
+// than 80 bits of security. SSL version 2 is prohibited. Any cipher suite using
+// MD5 for the MAC is also prohibited.
+//
+// Level 2
+// Security level set to 112 bits of security. As a result RSA, DSA and DH keys
+// shorter than 2048 bits and ECC keys shorter than 224 bits are prohibited. In
+// addition to the level 1 exclusions any cipher suite using RC4 is also
+// prohibited. SSL version 3 is also not allowed. Compression is disabled.
+//
+// AWS-LC's libssl doesn't support SSL version 2 or 3, and we have no support
```

Comment:
"AWS-LC's libssl doesn't support SSL version 2 or 3"

Unclear what this means.

Reply:
I took this from directly the OpenSSL documentation above, which is referring to the ancient SSL protocol. Would changing it to SSLv2 and SSLv3 be better?

Reply:
A'ha, yes that's better.